.gitpod.yml

```diff
@@ -1,5 +1,6 @@
 image: gitpod/workspace-postgres
 tasks:
+  # NOTE: As of 2021-1012, the contract helper DB seems to use the indexer DB so this likely won't work anymore
   - init: yarn && ./scripts/create_dev_dbs.sh
```

Comment:
It seems like this PR deletes this file, but still references it here in the `init` clause... can you confirm that we don't need `create_dev_dbs.sh` in the gitpod.yml file? AFAIK it is necessary to configure the Postgres gitpod instance.